math.go

```diff
@@ -378,6 +379,59 @@ var (
 			}
 			return valueNode(nil, "avg", Null, nil), nil
 		},
+		"b64decode": func(node *Node) (result *Node, err error) {
+			if node.IsString() {
+				if sourceString, err := node.GetString(); err != nil {
+					return nil, err
+				} else {
+					var result []byte
+					result, err = base64.StdEncoding.WithPadding(base64.StdPadding).DecodeString(sourceString)
+					if err != nil {
+						// then for NO_PAD encoded strings, if the first result with error
+						result, err = base64.StdEncoding.WithPadding(base64.NoPadding).DecodeString(sourceString)
+					}
+					if err != nil {
+						return nil, err
+					}
+					return valueNode(nil, "b64decode", String, string(result)), nil
+				}
+			}
+			return valueNode(nil, "b64decode", Null, nil), nil
+		},
+		"b64encoden": func(node *Node) (result *Node, err error) {
+			if node.IsString() {
+				if sourceString, err := node.GetString(); err != nil {
+					return nil, err
+				} else {
+					remainder := len(sourceString) % 3
+					size := len(sourceString) / 3 * 4
+					if remainder != 0 {
+						size += 4
```

Comment:
looks like it should be `+= 1 + remainder` instead of `+= 4`. 
And please check why it wasn't found by tests.
```suggestion
						size += 1 + remainder`
```

Reply:
Unfortunately, I forgot to run the tests after changes. They do fail locally because of `+= 4` .  I get following error
```
--- FAIL: TestFunctions2 (0.00s)
    --- FAIL: TestFunctions2/b64encode_no_padding_remainder_2 (0.00s)
        /Users/elena/Documents/Uni/Go/ajson/math_test.go:508: Wrong value: QSB0ZXN0IHN0cmluZw    --- FAIL: TestFunctions2/b64encode_no_padding_remainder_1 (0.00s)
        /Users/elena/Documents/Uni/Go/ajson/math_test.go:508: Wrong value: QSB0ZXN0IHN0cmluZy4FAIL
FAIL	github.com/spyzhov/ajson	0.335s
FAIL
```
After I changed it to `1 + remainder`, they don't fail anymore.